### DISCUSSION TITLE
Machine learning on Kubernetes

### DISCUSSION THREAD

**Discussion by johnugeorge** [cncf/maintainer-summit#2] [Maintainer Summit India 2024]
Kubeflow is one of the most popular MLOps platform on Kubernetes.

**Comment by palnabarun** [comment]
This session is accepted as an unconference. Please take notes in this document: https://docs.google.com/document/d/1MqroZh91JChZR4TFXGlHs3kx706T7CUJHJarV2_W1Tk/edit?usp=sharing
